subsys/bluetooth/controller/Kconfig

```diff
@@ -143,11 +143,24 @@ config BT_CTLR_SDC_MAX_CONN_EVENT_LEN_DEFAULT
 	  The time set aside for connections on every connection interval in
 	  microseconds. The event length and the connection interval are the
 	  primary parameters for setting the throughput of a connection.
+	  When BT_CTLR_SDC_CONN_EVENT_EXTEND_DEFAULT is set, more time may be used.
```

Comment:
sdc_hci_cmd_vs_conn_event_extend may be called by the application, as well.
I think it is not only about the BT_CTLR_SDC_CONN_EVENT_EXTEND_DEFAULT, but the question is if connecion event extension is enabled.